code/backend/pages/utilities/AzureSearchHandler.py

```diff
@@ -0,0 +1,66 @@
+from backend.pages.utilities.SearchHandlerBase import SearchHandlerBase
+from backend.batch.utilities.helpers.AzureSearchHelper import AzureSearchHelper
+import json
+import pandas as pd
+import streamlit as st
```

Comment:
I think _ideally_ we should try and avoild streamlit (and pandas?) from leaking outside of the pages. It makes this harder to reuse outside of the admin app.

Could we instead do something like create a `SearchResults` object that `perform_search` returns? It could then map what the search client returns to generic fields.

WDYT?